src/clj/clojure/core.clje

```diff
@@ -4582,10 +4582,10 @@
       (if load
         (try
           (load lib need-ns require)
-          (catch :error e :stack st
+          (catch :error e
```

Comment:
This was changed so that the bug in [ERL-1175](https://bugs.erlang.org/browse/ERL-1175) is not triggered.